components/combobox/index.css

```diff
@@ -0,0 +1,451 @@
+  --spectrum-combobox-block-spacing-edge-to-alert: var(--spectrum-field-top-to-alert-icon-medium);
+  --spectrum-combobox-spacing-edge-to-menu: var(--spectrum-component-to-menu-medium);
+  --spectrum-combobox-spacing-block-start-edge-to-textfield: var(--spectrum-component-top-to-text-100);
+  --spectrum-combobox-spacing-block-end-edge-to-textfield: var(--spectrum-component-bottom-to-text-100);
+  --spectrum-combobox-spacing-inline-start-edge-to-textfield: var(--spectrum-component-edge-to-text-100);
+  --spectrum-combobox-spacing-inline-end-edge-to-button: var(--spectrum-component-edge-to-text-100);
+}
+
+.spectrum-Combobox--sizeL {
+  --spectrum-combobox-block-size: var(--spectrum-component-height-200);
+  --spectrum-combobox-icon-size: var(--spectrum-workflow-icon-size-200);
+  --spectrum-combobox-font-size: var(--spectrum-font-size-200);
+
+  --spectrum-combobox-spacing-inline-icon-to-button: var(--spectrum-combo-box-visual-to-field-button-large);
+  --spectrum-combobox-block-spacing-edge-to-progress-circle: var(--spectrum-field-top-to-progress-circle-large);
+  --spectrum-combobox-block-spacing-edge-to-alert: var(--spectrum-field-top-to-alert-icon-large);
+  --spectrum-combobox-spacing-edge-to-menu: var(--spectrum-component-to-menu-large);
+  --spectrum-combobox-spacing-block-start-edge-to-textfield: var(--spectrum-component-top-to-text-200);
+  --spectrum-combobox-spacing-block-end-edge-to-textfield: var(--spectrum-component-bottom-to-text-200);
+  --spectrum-combobox-spacing-inline-start-edge-to-textfield: var(--spectrum-component-edge-to-text-200);
+  --spectrum-combobox-spacing-inline-end-edge-to-button: var(--spectrum-component-edge-to-text-200);
+}
+
+.spectrum-Combobox--sizeXL {
+  --spectrum-combobox-block-size: var(--spectrum-component-height-300);
+  --spectrum-combobox-icon-size: var(--spectrum-workflow-icon-size-300);
+  --spectrum-combobox-font-size: var(--spectrum-font-size-300);
+
+  --spectrum-combobox-spacing-inline-icon-to-button: var(--spectrum-combo-box-visual-to-field-button-extra-large);
+  --spectrum-combobox-block-spacing-edge-to-progress-circle: var(--spectrum-field-top-to-progress-circle-extra-large);
+  --spectrum-combobox-block-spacing-edge-to-alert: var(--spectrum-field-top-to-alert-icon-extra-large);
+  --spectrum-combobox-spacing-edge-to-menu: var(--spectrum-component-to-menu-extra-large);
+  --spectrum-combobox-spacing-block-start-edge-to-textfield: var(--spectrum-component-top-to-text-300);
+  --spectrum-combobox-spacing-block-end-edge-to-textfield: var(--spectrum-component-bottom-to-text-300);
+  --spectrum-combobox-spacing-inline-start-edge-to-textfield: var(--spectrum-component-edge-to-text-300);
+  --spectrum-combobox-spacing-inline-end-edge-to-button: var(--spectrum-component-edge-to-text-300);
+}
+
+.spectrum-Combobox {
+  /* Placeholders */
+  --spectrum-combobox-quiet-fieldbutton-border-radius: 0;
+  --spectrum-combobox-button-size: 32px;
```

Comment:
might need soft-coding. Also might not work for t-shirt sizing as the button inside the Combobox will be impacted by the outer t-shirt size.